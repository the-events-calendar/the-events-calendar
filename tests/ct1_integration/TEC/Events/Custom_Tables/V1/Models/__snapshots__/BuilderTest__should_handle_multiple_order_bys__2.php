<?php return 'SELECT *
FROM `wp_tec_occurrences`
ORDER BY `end_date_utc` DESC';
