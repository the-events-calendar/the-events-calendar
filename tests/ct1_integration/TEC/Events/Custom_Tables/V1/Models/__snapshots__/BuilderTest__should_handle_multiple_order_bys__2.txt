SELECT *
FROM `test_tec_occurrences`
ORDER BY `end_date_utc` DESC